docs/architecture/overview.md

```diff
@@ -1,61 +1,65 @@
 
-#### Accounts produce and consume notes to communicate
+[Transactions](transactions/overview.md) describe the production and consumption of notes by a single account. 
 
-![Architecture core concepts](../img/architecture/miden-architecture-core-concepts.gif)
+Executing a transaction always results in a STARK proof. 
 
-## State and Execution
-The state model captures all individual states of all accounts and notes. Finally, the execution model describes state progress in a sequence of blocks.
+The [transaction chapter](transactions/overview.md) describes the transaction design and implementation, including an indepth discussion of how transaction execution happens in the transaction kernel program.
 
-### State model
-[State](state.md) describes everything that is the case at a certain point in time. Individual states of accounts or notes can be stored onchain and offchain. This chapter describes the three different state databases in Miden.
+##### Accounts produce and consume notes to communicate
 
-### Execution model
-[Execution](execution.md) describes how the state progresses as aggregated state updates in batches, blocks, and epochs. This chapter describes the execution model and how blocks are built.
+![Architecture core concepts](../img/architecture/miden-architecture-core-concepts.gif)
 
-#### Operators capture and progress state
+## State and execution
 
-![Architecture state process](../img/architecture/miden-architecture-state-progress.gif)
+The actor-based execution model requires a radically different approach to recording the system's state. Actors and the messages they exchange must be treated as first-class citizens. Polygon Miden addresses this by combining the state models of account-based systems like Ethereum and UTXO-based systems like Bitcoin and Zcash.
 
-!!! info "Architecture tradeoffs"
-      Want to know more on why we designed Miden as is?
+Miden's state model captures the individual states of all accounts and notes, and the execution model describes state progress in a sequence of blocks.
 
-      **Polygon Miden's architecture**
-      
-      Polygon Miden’s architecture departs considerably from typical blockchain designs to support privacy and parallel transaction execution. In traditional blockchains state and transactions must be transparent to be verifiable. This is necessary for block production and execution. User generated zero-knowledge proofs allow state transitions, e.g. transactions, to be verifiable without being transparent.
+### State model
+
+[State](state.md) describes the state of everything at a certain point in time. Individual account or note states can be stored onchain and offchain. The [state chapter](state.md) describes the three different state databases in Miden.
```

Comment:
I think here you still use the sentence 

`[State](state.md) describes the state of everything at a certain point in time.`

which to me explains `state` with `state`. So I would prefer to have something like 

`[State](state.md) describes the condition of all accounts and notes at a certain point in time.`